Added git pill

Kardelio opened the pull request:
issue #21 

Also this is an awesome repo! Well done and Id love to contribute more!

Kardelio committed:
Added git pill

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -95,6 +95,17 @@ gitTypoPoop="
  _(_\ \)__
 (____\___))"
 
+gitTypoPill="
```
Here you do not need to escape characters. Just put the ascii art between simple quotes like this
```
'
 _________
[_________]
 |  .-.  |
 |,(o.o).|
 | >|n|< |
 |` `"` `|
 |POISON!|
 `"""""""`
'
```

Kardelio replied:
Oooooo didnt know that! thats awesome thanks for that! #alwaysLearning!

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -95,6 +95,17 @@ gitTypoPoop="
  _(_\ \)__
 (____\___))"
 
+gitTypoPill="
```
Please move this art just under `gitTypoExplosion7` so that they stay in alphabetical order

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -192,6 +203,15 @@ git() {
     curl -s http://whatthecommit.com/index.txt
     command git commit "${@:2}"
 
+  # pill
```
Please replace your comments with just `# pull`

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -192,6 +203,15 @@ git() {
+  # Made a horrible yellow and purple flashing
+  # message, enjoy!
+  elif [ "$1" = "pill" ]; then
+    echo -e "\033[93;105;5m---------------------------------\033[0m"
```
Nice one but we have nothing of color yet and I am not a fan of messing with user's colors. Could you just print the text without it ? Or was it part of the joke and I did not get it ?

Kardelio replied:
Understood! wil remove color and nah it was just me being excited and making it look crazy haha will remove it now

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -192,6 +203,15 @@ git() {
+    echo -e "\033[93;105;5m---------------------------------\033[0m"
+    echo -e "\033[93;105;5mNow thats a hard pill to swallow!\033[0m"
+    echo -e "\033[93;105;5m---------------------------------\033[0m"
+	echo "$gitTypoPill"
```
This line has a tab instead of 4 spaces. Please check that all your indentation is made of spaces.

Xaxetrov reviewed (commented):
Thanks for your contribution!

Kardelio committed:
Made changes suggested in the issue by repo owner

Kardelio commented:
@Xaxetrov updated the PR with a new commit with all your suggestions.
How is it now?
Thanks for your feedback btw!

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -192,6 +203,11 @@ git() {
     curl -s http://whatthecommit.com/index.txt
     command git commit "${@:2}"
 
+  # pull
```
Please move the section just above the "# remote" section so that sections stay ordered alphabetically.

Kardelio replied:
Will do hold on :D

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -192,6 +203,11 @@ git() {
+  elif [ "$1" = "pill" ]; then
+    echo "Now thats a hard pill to swallow!"
+    echo "$gitTypoExplosion7"
+
```
Do not forget to call the actual command, with arguments.

Kardelio replied:
Yeh totally forgot this and jsut saw it myself, hold on...

Kardelio committed:
Apologies forgot to add the command part of the function to trigger
the correct git pull script.
Also i was overwriting the explosionAsciiArt 7 and should have used 8
instead

Kardelio committed:
Moved section above remote secton

Kardelio commented:
@Xaxetrov I really appreciate you giving lots of great feedback on the PR, sorry i missed a bunch of things, but just submitted two more commits and hopefully they are ok now?
Thanks again I love the attention to detail and honest feedback! Very cool! Again please let me know if there are any more things that need dealing with

Also is it possible to squash my commits into a single one, if and when, the PR gets accepted?

Xaxetrov commented on shellGitTypo.sh:
```diff
@@ -87,6 +87,16 @@ gitTypoExplosion7='
      ) /
     ,---.'
     
+gitTypoExplosion8='
```
Why did you change the name ?
gitTypoPill was OK.

I was maybe not very clear in a previous comment, but I was just complaining about keeping constant names in order.

Kardelio replied:
Ahhhh sorry yeah i got a little confused with that, i have now changed the name back to how it was, sorry for my confusion, my bad

Xaxetrov reviewed (commented):
Just one small pb left.

I will squash the commits when merging.
You are welcome to contribute more ! You can:
- Pick other open issues
- Create new ones if you have ideas
- Directly do other PRs if you have ideas and want to do them yourself right away.

Kardelio committed:
changed name back
